manual/core/compression/README.md

```diff
@@ -19,8 +19,11 @@ Compression must be set before opening a session, it cannot be changed at runtim
 
 
 Two algorithms are supported out of the box: [LZ4](https://github.com/jpountz/lz4-java) and
-[Snappy](http://google.github.io/snappy/). Both rely on third-party libraries, declared by the
-driver as *optional* dependencies; if you enable compression, you need to explicitly depend on the
+[Snappy](http://google.github.io/snappy/). The LZ4 implementation is a good
+first choice; it offers fallback implementations in case native libraries fail
+to load and benchmarks suggest it offers better performance and compression
+ratios over Snappy. Both rely on third-party libraries, declared by the driver
+as *optional* dependencies; if you enable compression, you need to explicitly depend on the
```

Comment:
Maybe we can do similar perf tests to those: `http://java-performance.info/performance-general-compression/`
and point users to our perf tests results? If the user will see charts with a performance comparison of our driver using different compression algorithms, it will give the users actual insight why LZ4 is better

Reply:
I think it would be adequate to just refer to that blog post for further reading, i'll add that link.   My intent here is just to make it easy for users to make a choice without having to think about it much.  At this point there isn't really a reason to adopt Snappy over LZ4 unless you have a really good reason, so LZ4 should be the direction we point people in.